src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -418,14 +444,19 @@ function FloatingActionButtonAndPopover({onHideCreateMenu, onShowCreateMenu}: Fl
                                   icon: Expensicons.InvoiceGeneric,
                                   text: translate('workspace.invoices.sendInvoice'),
                                   onSelected: () =>
-                                      interceptAnonymousUser(() =>
+                                      interceptAnonymousUser(() => {
```

Comment:
@ishpaul777 Can you verify this change working on your iOS native simulator?


https://github.com/user-attachments/assets/afb3d38a-4393-4bfc-a011-2f2d3132610a